packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -258,6 +258,7 @@ export class FASTCombobox extends FormAssociatedCombobox {
             ) as FASTListboxOption | null;
 
             if (!captured || captured.disabled) {
+                this.open = !this.open;
```

Comment:
`this.open = false`

This will always be `false` so clearer to just set it without evaluating.